extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/handler/DimmerThingHandler.java

```diff
@@ -0,0 +1,328 @@
+        Configuration configuration = getConfig();
+        if (configuration.get(CONFIG_DMX_ID) != null) {
+            DmxBridgeHandler bridgeHandler = (DmxBridgeHandler) getBridge().getHandler();
+            try {
+                List<BaseChannel> configChannels = BaseChannel.fromString((String) configuration.get(CONFIG_DMX_ID),
+                        bridgeHandler.getUniverseId());
+                logger.trace("found {} channels in {}", configChannels.size(), this.thing.getUID());
+                for (BaseChannel channel : configChannels) {
+                    channels.add(bridgeHandler.getDmxChannel(channel, this.thing));
+                }
+            } catch (IllegalArgumentException e) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, e.getMessage());
+                return;
+            }
+            if (configuration.get(CONFIG_DIMMER_FADE_TIME) != null) {
+                fadeTime = ((BigDecimal) configuration.get(CONFIG_DIMMER_FADE_TIME)).intValue();
+                logger.trace("setting fadeTime to {} ms in {}", fadeTime, this.thing.getUID());
+            }
+            if (configuration.get(CONFIG_DIMMER_DIM_TIME) != null) {
+                dimTime = ((BigDecimal) configuration.get(CONFIG_DIMMER_DIM_TIME)).intValue();
+                logger.trace("setting dimTime to {} ms in {}", fadeTime, this.thing.getUID());
+            }
+            if (configuration.get(CONFIG_DIMMER_TURNONVALUE) != null) {
+                String turnOnValueString = String.valueOf(fadeTime) + ":"
+                        + ((String) configuration.get(CONFIG_DIMMER_TURNONVALUE)) + ":-1";
+                ValueSet turnOnValue = ValueSet.fromString(turnOnValueString);
+                if (turnOnValue != null) {
+                    this.turnOnValue = turnOnValue;
+                    logger.trace("set turnonvalue to {} in {}", turnOnValue, this.thing.getUID());
+                } else {
+                    logger.warn("could not set turnonvalue to {} in {}, malformed", turnOnValueString,
+                            this.thing.getUID());
+                }
+            }
+            if (configuration.get(CONFIG_DIMMER_TURNOFFVALUE) != null) {
+                String turnOffValueString = String.valueOf(fadeTime) + ":"
+                        + ((String) configuration.get(CONFIG_DIMMER_TURNOFFVALUE)) + ":-1";
+                ValueSet turnOffValue = ValueSet.fromString(turnOffValueString);
+                if (turnOffValue != null) {
+                    this.turnOffValue = turnOffValue;
+                    logger.trace("set turnoffvalue to {} in {}", turnOffValue, this.thing.getUID());
+                } else {
+                    logger.warn("could not set turnoffvalue to {} in {}, malformed", turnOffValueString,
+                            this.thing.getUID());
+                }
+            }
+            scheduler.schedule(new Runnable() {
```

Comment:
> It seems to be impossible to edit the thing during initialization.

I would expect it to be possible. If you indeed run into the 5000ms exception in that case, please open an issue so that this can be addressed in the framework.